udata/core/dataservices/models.py

```diff
@@ -132,13 +131,29 @@ def __str__(self):
     )
     description = field(db.StringField(default=""), description="In markdown")
     base_api_url = field(db.URLField(), sortable=True)
-    endpoint_description_url = field(db.URLField())
+
+    endpoint_description_url = field(
+        db.URLField()
+    )  # Legacy: to remove / replaced by `machine_documentation_url` or `technical_documentation_url`
```

Comment:
Don't you want to remove legacy fields this PR since we will have the migration that will clear these?

Reply:
I thought we needed to keep these fields at least for the migration? To not fail the queries?

Reply:
I just check running the migration after removing the old fields, it fails with "Cannot resolve field "is_restricted"

Reply:
You would need to access the collection directly as in [this migration for example](https://github.com/opendatateam/udata/blob/master/udata/migrations/2024-03-22-migrate-activity-kwargs-to-extras.py) instead of using mongoengine documents.

Reply:
Done in bba5d7383, it wasn't easy but it's way better (it should work even if we run the migrations far in the future)